agent/lib/kontena/websocket_client.rb

```diff
@@ -171,6 +172,7 @@ def on_close(event)
         handle_invalid_version
       end
       info "connection closed with code #{event.code}"
+      notify_actors('websocket:close', nil)
```

Comment:
what's the difference from other actors point-of-view between `websocket:close` and `websocket:disconnect`? I think in both cases they should stop trying to send data to master, right?